src/core/subdirdata.cpp

```diff
@@ -197,16 +235,16 @@ namespace mamba
     {
         if (m_target->result != 0 || m_target->http_status >= 400)
         {
-            LOG_INFO << "Unable to retrieve repodata (response: " << m_target->http_status
-                     << ") for " << m_repodata_url;
+            LOG_DEBUG << "Unable to retrieve repodata (response: " << m_target->http_status
```

Comment:
why is this log debug? It seems INFO would be appropriate?

Reply:
humm yeah maybe it's a warning here